ui/src/services/kolay/selected.ts

```diff
@@ -25,6 +26,37 @@ import type RouterService from '@ember/routing/router-service';
 
 const firstPath = '/1-get-started/intro.md';
 
+/**
+ * Like an ember-concurrency task,
+ * if we ignore concurrency and only care about the states
+ * (and want automatic invocation based on derivation)
+ */
+class MDRequest {
+  constructor(private urlFn: () => string) {}
+
+  @use last = RemoteData<string>(this.urlFn);
```

Comment:
this needs to also check if the page is in the manifest -- else Vercel and Cloudflare will serve us a valid HTML page (as is common for SPAs)

I'll need to copy some of the logic from this other PR: https://github.com/universal-ember/kolay/pull/82/files#diff-2d77d612362980f19e6d9c6943418b5a73659546fe32510cabadb74baa0d12afR38